installers/ansible/roles/pgo-operator/files/pgo-configs/cluster-deployment.json

```diff
@@ -282,11 +279,11 @@
                       {{ end }}
                     {{ end }}
                     {
-                        "name": "recover-volume",
-                        "emptyDir": { "medium": "Memory" }
-                    }, {
                         "name": "report",
-                        "emptyDir": { "medium": "Memory" }
+                        "emptyDir": {
+                          "medium": "Memory",
+                          "sizeLimit": "64Mi"
```

Comment:
As I understand it, this will cause the pod to be evicted if it uses more than this amount: https://issue.k8s.io/63126

It looks like Kubernetes 1.20 has an alpha feature to finally consider this amount when creating the volume: https://pr.k8s.io/94444

- https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1967-size-memory-backed-volumes

Reply:
From my understanding, it would evcit the Pod. But it sounds like there are restrictions that if we don't set a size limit, it will not schedule the Pod.

We could set the sizeLimit to be a higher value. That said, I had mainly tested generating pgBadger files to see how large they were. From an initial run, it was ~1MB. From my experience with pgBadger, the assets around it ar enot that large -- the input (PostgreSQL logs) tends to be the large bit.

Reply:
> From my experience with pgBadger, the assets around it ar enot that large -- the input (PostgreSQL logs) tends to be the large bit.

Good point.